specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -104,6 +110,11 @@ export function Header({
             ? 'dark:border-l'
             : 'dark:border-t'
         }
+        ${
+          isSideBarLight === 'matchThemeColor' && !isDarkMode
+            ? 'bg-gray-100 shadow-md shadow-gray-400'
```

Comment:
(optional) could you double check that these light mode styles are the ones that we had for menu in light mode before it got permanently switched to dark mode?
if they are not the same, make sure that was intentional. if they are the same, then great

Reply:
I took the style from how it was before :+1: